### ISSUE TITLE
Option to let affiliate pick weather they want to be paid through bank transfer or PayPal

### ISSUE THREAD

**Issue by jjclinton** [AffiliateWP/AffiliateWP#2545]
I noticed that there isn't an option to let the administrator say if he or she wants to let affiliates choose their payout method. AffiliateWP does support bank transfer, but you'll have to communicate it to your affiliates in someway. This I think, is not very user-friendly. I'd like to automate this and let the affiliate choose weather they want to get paid via bank transfer or PayPal, but only if the website's admin says affiliates can choose this. 

The admin should then also be able to select bank transfer or PayPal as a default payment method.

To summarize:
**Administrator options**

1. Select PayPal as payment method
2. Select bank transfer as payment method
3. Let the affiliate select their desired payment method
